pallets/ethereum-xcm/src/lib.rs

```diff
@@ -232,6 +232,16 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
+	fn transaction_len(transaction: &Transaction) -> u64 {
+		transaction
+			.encode()
+			.len()
+			// pallet index
+			.saturating_add(1)
```

Comment:
```suggestion
			.saturating_add(2)
```

please merge the 2 saturating instructions (and comments)